### DISCUSSION TITLE
UCB CS188: Introduction to Artificial Intelligence - csdiy.wiki

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#555] [Announcements]
# UCB CS188: Introduction to Artificial Intelligence - csdiy.wiki

CS自学指南

https://csdiy.wiki/en/%E4%BA%BA%E5%B7%A5%E6%99%BA%E8%83%BD/CS188/

**Comment by AAAAyl0n** [comment]
CS188的gradescope的entry code好像没法用了

**Reply by expecto347** [reply]
可以用啊，学校选UC Berkeley

**Comment by DiegoEme** [comment]
I am not able to take this course as a Berkeley account is needed to access the website, videos and assignments :/
Maybe you can replace this course with MIT's Artificial Intelligence https://ocw.mit.edu/courses/6-034-artificial-intelligence-fall-2010/
